Test/DafnyTests/RunAllTestsOption.dfy.expect

```diff
@@ -12,7 +12,7 @@ Failing2: FAILED
 	RunAllTestsOption.dfy(29,2): expectation violation
```

Comment:
Can we avoid the changes in RunAllTestsOption.dfy.expect and in contract-wrappers?